### DISCUSSION TITLE
Want a Galaxy Badge📛

### DISCUSSION THREAD

**Discussion by samhackathon** [community/community#34079] [Other Feature Feedback, Questions, & Ideas] [answered]
Anyone can tell me about the Galaxy Badge

**Comment by ossamamehmood** [comment, marked as answer]
@samhackathon to achieve the 'Galaxy Brain' achievement, take note of the following:

Answer at least 2 discussions.

- Take note, you CAN NOT mark your answers to your questions. They have to be marked by another user.
- Discussions should be in a public repository.
- There's a current bug observed by some users whereby marking your answers can prevent you from getting the badge.

I hope this answer has helped you, and all the best in achieving the achievement! :)

**Comment by github-staff** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
